asf_core_data/notebooks/get_address_matching_sample.py

```diff
@@ -0,0 +1,318 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.15.2
+#   kernelspec:
+#     display_name: asf_core_data_kernel
+#     language: python
+#     name: asf_core_data_kernel
+# ---
+
+# %%
+import pandas as pd
+import os
+from datetime import datetime
+
+# %%
+from asf_core_data.pipeline.mcs.process import mcs_epc_joining as joining
+
+# %% [markdown]
+# ## Set local_core_data_dir where EPC data is stored
+
+# %%
+## Set your local ASF core data dir
+local_core_data_dir = ""
+
+## Output data will be saved here
+outputs_data_dir = os.path.abspath("../../outputs/data")
```

Comment:
If you do it in the following way, the output will always be saved in the same place, independently of where you're running it from.

And because the folder data doesn't exist, we need to create it.
```suggestion
outputs_data_dir = os.path.join(PROJECT_DIR,"outputs/data")
os.makedirs(outputs_data_dir, exist_ok=True)
```